README.md

```diff
@@ -48,42 +59,48 @@ The script will generate solar forecasts at the specified intervals between the
 
 The source code is currently hosted on GitHub at: https://github.com/openclimatefix/Open-Source-Quartz-Solar-Forecast
 
-Binary installers for the latest released version are available at the Python Package Index (PyPI)  
+Binary installers for the latest released version are available at the Python Package Index (PyPI)
+
 ```bash
 pip install quartz-solar-forecast
 ```
+
 You might need to install the following packages first
+
 ```bash
 conda install -c conda-forge pyresample
-````
-This can solve the [bug: ___kmpc_for_static_fini](https://github.com/openclimatefix/Open-Source-Quartz-Solar-Forecast/issues/32).
+```
+
+This can solve the [bug: \_\_\_kmpc_for_static_fini](https://github.com/openclimatefix/Open-Source-Quartz-Solar-Forecast/issues/32).
 
 ## Model
 
 The model is a gradient boosted tree model and uses 9 NWP variables.
```

Comment:
I think we can update this section too.
I would mention what our model does (feature list, XGBoost, hyperparameter tuning), a general error comparison with the current OCF model (ie overall MAE Tryo vs OCF and maybe the monthly plot) and a link to the blogpost for further details.

Reply:
I updated the Readme. I removed quite a lot because it does not apply to the new model / analysis. Please feel free to comment on what you think is missing.